guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -485,12 +485,13 @@ If you deploy {Project} with the {SmartProxy} templates feature, you must config
 
 For security reasons, {Project} recognizes this HTTP header only from localhost.
 For each individual {SmartProxy}, you must configure a regular expression to recognize hosts' IP addresses.
-From the {ProjectWebUI}, you can do this by navigating to *Administer* > *Settings* > *Provisioning*, and changing the *Remote address* setting.
-From the CLI, you can do this by entering the following command:
+Previously, this was done in *Administer* > *Settings* > *Provisioning*, using the *Remote address* setting.
```

Comment:
Do we even need to say this? We can just include where to do it as only this version counts.

Reply:
Sure, I also sligthly reworded this. How about that?